homeassistant/components/here_weather/weather.py

```diff
@@ -0,0 +1,222 @@
+"""Weather platform for the HERE Destination Weather service."""
```

Comment:
It would be better to save this platform for a later PR.

Reply:
balloob told me to add it: https://github.com/home-assistant/core/pull/28910#discussion_r561017430

Reply:
I know, but it's not about user features but about splitting the review work for both contributor and reviewers. We'd add this platform next as a follow up.